src/components/atoms/shared/button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import { Light, StyledButton } from './Buttonstyles';
+
+function Button({ method = undefined }) {
+	return (
+		<>
+			<StyledButton>button</StyledButton>
```

Comment:
Make button such that it can take things such as type, link and children and other things.